.github/workflows/_buildpacks-release.yml

```diff
@@ -278,6 +284,13 @@ jobs:
     needs: [compile]
     runs-on: ${{ inputs.ip_allowlisted_runner }}
     steps:
+      - name: Get token for GH application (Linguist)
```

Comment:
Is there a reason for moving this step? If not, maybe leave it where it was since it just seems like noise for this PR.

Reply:
I moved the step to before checkout, like the other jobs, but it sounds like that is unnecessary.